src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytesAsync.cs

```diff
@@ -85,6 +85,21 @@ public Task ReadFileOver2GBAsync()
             return Assert.ThrowsAsync<IOException>(async () => await File.ReadAllBytesAsync(path));
         }
 
+        [Fact]
+        [OuterLoop]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/45954", TestPlatforms.Browser)]
+        public Task MaxArrayLengthAsync()
```

Comment:
"ReadFileOverMaxArrayLengthAsync" for consistency with the sync test name?  Just wondering why they're named differently.